test/integration/world_with_initial_sim_time_from_cli.cc

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <string>
+#include "gazebo/common/Time.hh"
+#include "gazebo/test/ServerFixture.hh"
+
+using namespace gazebo;
+
+const double initialSimTime = 1675117690.123456;
+
+/// \brief Helper class that initializes each test.
+class WorldWithInitialSimTimeFromCliTest : public ServerFixture
+{
+  /// \brief Class constructor.
+  public: WorldWithInitialSimTimeFromCliTest()
+  {
+    // Start the server with a log file and paused.
```

Comment:
```suggestion
    // Start the server with an initial sim time and paused.
```

Reply:
Fixed in https://github.com/gazebosim/gazebo-classic/pull/3294/commits/5078e9d623e034301cc897cc4395ebe31bd26ea0.